keybase/platform_windows.go

```diff
@@ -191,12 +199,13 @@ func (c context) Apply(update updater.Update, options updater.UpdateOptions, tmp
 		return fmt.Errorf("No asset")
 	}
 	var args []string
-	auto, _ := c.config.GetUpdateAuto()
-	canBeSilent, _ := CheckCanBeSilent(update.Asset.LocalPath, c.log, CheckRegistryUninstallCode)
-	if auto && canBeSilent {
+	_, auto := c.config.GetUpdateAuto()
+	if auto && !c.config.GetUpdateAutoOverride() {
+
 		args = append(args, "/quiet")
 	}
-	_, err := command.Exec(update.Asset.LocalPath, args, time.Hour, c.log)
+	// We really don't want to kill installers, hence 30 days here
+	_, err := command.Exec(update.Asset.LocalPath, args, time.Hour*24*30, c.log)
```

Comment:
...but there is another thing we have to deal with: the person can exit with a reboot pending, in which case the updater also shouldn't re-launch. Still thinking about what to do about that.